runelite-client/src/main/java/net/runelite/client/game/FishingSpot.java

```diff
@@ -171,12 +171,13 @@
 	DARK_CRAB("Dark Crab", ItemID.RAW_DARK_CRAB,
 		FISHING_SPOT_1536
 	),
-	COMMON_TENCH("Common tench, Bluegill, Greater siren, Mottled eel", ItemID.COMMON_TENCH,
+	COMMON_TENCH("Common tench, Bluegill, Greater siren, Mottled eel", "Common tench", ItemID.COMMON_TENCH,
 		FISHING_SPOT_8523);
 
 	private static final Map<Integer, FishingSpot> SPOTS;
 
 	private final String name;
+	private final String highestLevelFish;
```

Comment:
This doesn't always represent the highest level fish, so either fix the entries for which that's not the case or change the name.